### ISSUE TITLE
Create a basic schema for MySQL Database

### ISSUE THREAD

**Issue by Deepakraj8055** [amosproj/amos2022ws01-firmware-scraper#29]
## User story
1. As a user when i am using the amos firmware scraper
2. I want the output must have the following mandatory fields/attributes:

- Manufacturer (e.g., Siemens, D-Link, AVM, ABB, ...)
- Model/Name (e.g., S7-1500, DIR300, Scalance-X, ...)
- Version (e.g., 1.2.3)
- Type (e.g., Router, Switch, Firewall, PLC, ...)
- Release Date (if available)
- Checksum (sha512)
- EMBA tested (yes/no)
- EMBA link to report (filesystem)
- EMBArk link to report (http://...)
- Firmware download link (vendor link)
- Firmware filesystem link (to find it on the filesystem)

3. So that we have a standardised template

## Acceptance criteria
* All the above mentioned fields/attributes must be visible in the output

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
